mssql-mixin/dashboards_out/mssql_overview.json

```diff
@@ -192,7 +192,7 @@
                      "type": "prometheus",
                      "uid": "${prometheus_datasource}"
                   },
-                  "expr": "increase(mssql_kill_connection_errors_total{job=~\"integrations/mssql\",job=~\"$job\",cluster=~\"$cluster\",instance=~\"$instance\"}[$__rate_interval:])",
+                  "expr": "rate(mssql_kill_connection_errors_total{job=~\"integrations/mssql\",job=~\"$job\",cluster=~\"$cluster\",instance=~\"$instance\"}[$__rate_interval])",
```

Comment:
some expr got changed function from increase to rate at the end. Just to check that this is what you want. If not, you can change rateFunction to be increase for some signals. See Signals README.

Reply:
Thanks for the catch 👍 should be resolved now